src/app/asset-upload/asset-upload.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="asset-upload">
+    <mat-expansion-panel id="asset-upload-panel">
+        <mat-expansion-panel-header>
+            <mat-panel-title>
+                Asset Upload
+            </mat-panel-title>
+        </mat-expansion-panel-header>
+        <button mat-raised-button 
+            color="primary" 
+            type="button" 
+            (click)="fileInput.click()">
+            <span>Upload</span>
+            <input #fileInput 
+                type="file" 
+                (change)="onFileInput($event)" 
+                style="display:none;" 
+                name="filename"
+                accept="image/*,video/*"/>
+        </button>
+        <p *ngIf="file; else elseBlock">{{file.name}}</p>
```

Comment:
The else isn't mandatory, so we can remove it and the elseBlock since it's empty.